impl/src/main/java/dao/QuestionDao.java

```diff
@@ -64,13 +145,17 @@
             ":question.question, " +
             ":question.title, " +
             ":question.bounty, " +
-            "0, " +
             "NOW(), " +
             ":userId, " +
             ":question.slackId" +
             ")")
     Observable<GeneratedKey<Long>> addQuestion(long userId, Question question);
 
+    @Update("UPDATE question " +
+        "SET created_at=:createdAt " +
+        "WHERE question.id=:question.id")
+    Observable<Integer> setCreatedAt(Question question, LocalDateTime createdAt);
```

Comment:
What is the usecase of updating created_at?

Reply:
Testing sort order.

Reply:
Is this method due to my comment here? I see now that I failed to mention that I meant a separate DAO class for testing, sorry! 

https://github.com/FortnoxAB/rocket-fuel/pull/64#discussion_r313726412

Reply:
Is this method due to my comment here? I see now that I failed to mention that I meant a separate DAO class for testing, sorry! 

https://github.com/FortnoxAB/rocket-fuel/pull/64#discussion_r313726412